pkg/odo/cli/config/set.go

```diff
@@ -101,9 +101,16 @@ func (o *SetOptions) Run() (err error) {
 
 	if !o.configForceFlag {
 		if isSet := o.lci.IsSet(o.paramName); isSet {
-			if !ui.Proceed(fmt.Sprintf("%v is already set. Do you want to override it in the config", o.paramName)) {
-				fmt.Println("Aborted by the user.")
-				return nil
+			if strings.ToLower(o.paramName) == "name" || strings.ToLower(o.paramName) == "project" || strings.ToLower(o.paramName) == "application" {
+				if !ui.Proceed(fmt.Sprintf("You are changing the component's %s. This action might result in the creation of a duplicate component. If your component is already pushed, please delete the component %s after you apply the changes (odo component delete %s --app %s --project %s)", o.paramName, o.lci.GetName(), o.lci.GetName(), o.lci.GetApplication(), o.lci.GetProject())) {
```

Comment:
Does it make sense to change `please delete the component %s after you ` to `please delete the component %q after you ` for readability?

Reply:
Fixed :+1: